.travis.yml

```diff
@@ -45,6 +45,9 @@ matrix:
     - env:
       - MVN_TESTS='-Dtest=TestLocalBinarySpilledQueries,TestPostgreSqlDistributedQueries,TestMySqlDistributedQueries'
       - MVN_MODULES='-pl presto-tests,presto-mysql,presto-postgresql'
+    - env:
+      - PRODUCT_TESTS='-g hdfs_no_impersonation,authorization'
```

Comment:
I don't think that `authorization` is actually the group you would like to run. `authorization` is the group that contains `GRANT/REVOKE` tests. Instead of that you should rather run `storage_formats,cli`, as we do for `singlenode-kerberos-hdfs-impersonation` in main `.travis.yaml`.